vendor/vendor-ios/src/main/kotlin/com/malinskiy/marathon/ios/DockerTestParser.kt

```diff
@@ -24,20 +24,18 @@ class BinaryTestParser(private val binaryParserDockerImageName: String): DockerT
 
     override fun listTests(testRunnerPaths: List<File>): List<String> {
         val pathMappings = testRunnerPaths.map {
-            it.absolutePath to File("/tmp").resolve(it.name).absolutePath }
+            it.absolutePath to File("/tmp").resolve(it.name).absolutePath
+        }
 
         val command =
-                listOf("docker", "run", "--rm") +
-                    pathMappings.map {
-                    listOf("-v", "${it.first}:${it.second}:ro") }
-                        .flatten() + binaryParserDockerImageName + pathMappings.map { it.second }
-
-        val builder = ProcessBuilder()
-                .command(command)
-                .directory(File(System.getProperty("user.dir")))
```

Comment:
why did we remove this directory as working folder?

Reply:
that's only about a docker process and it could be run from anywhere, so I see no reason in specifying this dir here.